### PR TITLE
Add basic functions for posting

### PR DESCRIPTION
**Pull request by MarieEngel**
Please have a a look :)

### CODE REVIEW

File: post/models.py [hunk 1]
```diff
@@ -1,3 +1,27 @@
+from sqlite3 import Timestamp
```
Comment (SyncopatedPandemonium) [inline post/models.py:1]: Do we need this import?

> from sqlite3 import Timestamp

File: post/forms.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+from django import forms
+from django.forms import HiddenInput, ModelForm
```
Comment (SyncopatedPandemonium) [inline post/forms.py:2]: Do we need ` HiddenInput`

Comment (MarieEngel) [reply post/forms.py:2]: No, not for now,  can be deleted

File: post/models.py (hunk 1 above)
Comment (MarieEngel) [reply post/models.py:1]: no, was only for trying out

File: post/templates/post/home.html [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{% extends 'post/base.html' %}
+{% block title %}Latest Posts{% endblock %}
```
[the remark this answers is not in the document]
Comment (MarieEngel) [reply post/templates/post/home.html:2]: no, we don't need to specify it in base.html, just delete it :)

[the remark this answers is not in the document]
Comment (SyncopatedPandemonium) [reply post/templates/post/home.html:2]: Yes, I will do it today.

**Review by SyncopatedPandemonium — CHANGES_REQUESTED** [review]
Maybe we can remove the pictures?
Except from that with really small changes it looks great:)

File: post/models.py (hunk 1 above)
Comment (MarieEngel) [reply post/models.py]: Done

File: post/forms.py (hunk 2 above)
Comment (MarieEngel) [reply post/forms.py]: Done

**Review by SyncopatedPandemonium — COMMENTED** [review]
Looks good now:)

**Review by SyncopatedPandemonium — APPROVED** [review]
Looks good:)

**Pull request merged**